herepy/public_transit_api.py

```diff
@@ -370,34 +360,20 @@ def coverage_nearby(self, details, center):
                 'apikey': self._api_key}
         return self.__get(data, 'coverage/nearby.json', 'LocalCoverage')
 
-    def route_excluding_changes_transfers(self,
-                                          departure,
-                                          arrival,
-                                          time,
-                                          routing_type=PublicTransitRoutingType.time_tabled,
-                                          changes=-1):
-        """Request a direct public transit route excluding changes and transfers.
-        Args:
-          departure (array):
-            array including latitude and longitude in order.
-          arrival (array):
-            array including latitude and longitude in order.
-          time (str):
-            time formatted in yyyy-mm-ddThh:mm:ss.
-          routing_type (PublicTransitRoutingType):
-            type of routing. Default is time_tabled.
-          changes (int):
-            Maximum number of changes or transfers. Default is -1 and max is 6.
-        Returns:
-          PublicTransitResponse
-        Raises:
-          HEREError
-        """
-
-        data = {'dep': str.format('{0},{1}', departure[0], departure[1]),
-                'arr': str.format('{0},{1}', arrival[0], arrival[1]),
-                'time': time,
-                'apikey': self._api_key,
-                'routing': routing_type.__str__(),
-                'changes': changes}
-        return self.__get(data, 'route.json', 'Connections')
+def _get_response_with_short_route(public_transit_response):
```

Comment:
Is there any specific reason that you add this method out of class scope?

Reply:
It just felt weird having it access `self.`

I can change it if you want me to.

Reply:
I think it's better if you add inside the class.